modules/Android.mk

```diff
@@ -0,0 +1 @@
+include $(call all-named-subdir-makefiles,$(GAPPS_PRODUCT_PACKAGES))
```

Comment:
this requires that the module names exactly match the directory names.  I think this true now, although it wasn't always like that.